corehq/apps/userreports/templates/userreports/edit_data_source.html

```diff
@@ -4,6 +4,11 @@
 {% load i18n %}
 
 {% block page_content %}
+  {% if data_source.is_rebuild_in_progress %}
+        <div id="built-warning" class="alert alert-warning">
+          <h4><i class="fa fa-exclamation-triangle"></i> {% trans "Please note that this datasource is being rebuilt." %}</h4>
+        </div>
+  {% endif %}
```

Comment:
with this here do we still want the other notices:

https://github.com/dimagi/commcare-hq/blob/942d3e7db3147f013ada2f9eb557e5b95c66ff91/corehq/apps/userreports/templates/userreports/edit_data_source.html#L91-L97

https://github.com/dimagi/commcare-hq/blob/942d3e7db3147f013ada2f9eb557e5b95c66ff91/corehq/apps/userreports/templates/userreports/edit_data_source.html#L52-L57